apophis/cmds/remind.py

```diff
@@ -1,25 +1,61 @@
 """ Reminder Command """
 from cmds.command import Command
-# import asyncio
-# import time
+import asyncio
 import json
 import logging
+import time
+from datetime import datetime
 
 
-units_to_seconds = {
-    "minute" : 60, 
-    "hour" : 3600, 
-    "day" : 86400, 
-    "week": 604800, 
-    "month": 2592000
-}
+def convert_times(when):
+    t = {
+        "year": int(datetime.now().strftime("%Y")),
+        "month": int(datetime.now().strftime("%m")),
+        "day": int(datetime.now().strftime("%d")),
+        "hour": int(datetime.now().strftime("%H")),
+        "minute": int(datetime.now().strftime("%M")),
+    }
+
+    iau = [
+        {"int": int(w.split()[0]), "unit": w.split()[1]}
+        for w in when
+    ]
+
+    for i in range(len(iau)):
+        if iau[i]['unit'].startswith('y'):
+            t['year'] = int(iau[i]['int'] + t['year'])
+        elif iau[i]['unit'].startswith('mo'):
+            t['month'] = int(iau[i]['int'] + t['month'])
+        elif iau[i]['unit'].startswith('d'):
+            t['day'] = int(iau[i]['int'] + t['day'])
+        elif iau[i]['unit'].startswith('h'):
+            t['hour'] = int(iau[i]['int'] + t['hour'])
+        elif iau[i]['unit'].startswith('mi'):
+            t['minute'] = int(iau[i]['int'] + t['minute'])
+
+    
+    time_string = "{0}-{1}-{2} {3}:{4}".format(
+        t['year'],
+        t['month'],
+        t['day'],
+        t['hour'],
+        t['minute']
+    )
```

Comment:
You don't have to create a `datetime` from a string, it has a constructor that you can pass those time periods into.